nicolalandro_ntsnet-cub200_ntsnet.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: ntsnet
+summary: a fine grane model for image classification.
+image: nts-net.png
+author: Moreno Carraffini and Nicola Landro
+tags: [vision]
+github-link: https://github.com/nicolalandro/ntsnet-cub200/archive/master.zip
+featured_image_1: no-image
+featured_image_2: no-image
+accelerator: "cuda-optional"
+---
+
+```python
+# Preprocessing image of cube
+# transform_train = transforms.Compose([
+#     transforms.Resize((600, 600), Image.BILINEAR),
+#     transforms.CenterCrop((448, 448)),
+#     transforms.RandomHorizontalFlip(),  # solo se train
+#     transforms.ToTensor(),
+#     transforms.Normalize((0.485, 0.456, 0.406), (0.229, 0.224, 0.225)),
+# ])
+import torch
+model = torch.hub.load('nicolalandro/ntsnet-cub200', 'ntsnet', pretrained=True, **{'topN': 6, 'device':'cpu', 'num_classes': 200})
+# top_n_coordinates, concat_out, raw_logits, concat_logits, part_logits, top_n_index, top_n_prob = model(img)
```

Comment:
Would you mind adding an example about how the inference works on real bird picture?

Reply:
I want to add also a better part of documentation in order to make easy to train it yourself. But now I do not have much time. The idea is to share it quickly with this small documentation and add some better explanation and sample in the future. Do you think it is possible?